handbook/product-design/product-design.rituals.yml

```diff
@@ -16,7 +16,7 @@
   task: "Design sprint kickoff" # 2024-03-06 TODO: Link to responsibility or corresponding "how to" info e.g. https://fleetdm.com/handbook/company/product-groups#making-changes
   startedOn: "2024-03-07" 
   frequency: "Triweekly" 
-  description: "Create stories for feature requests prioritized during Feature fest and add these stories to the Drafting board, add stories to the roadmap chart, assign stories to Product Designers, align on priorities, and create upcoming reference docs release branch." 
+  description: "Create stories (and, as needed, engineering research stories) for feature requests prioritized during Feature fest; add these stories to the Drafting board and the roadmap chart; assign Product Designers (PD) and Engineering Support Individuals (ESP) to stories; align on priorities; and create an upcoming reference docs release branch."
```

Comment:
```suggestion
  description: "Review stories (and, as needed, create engineering research stories) for feature requests prioritized during Feature fest; assign Engineering DRIs to stories; align on priorities; and create an upcoming reference docs release branch."
```

Reply:
> I believe we talked about creating the stories asynch after Feature fest

@eugkuo that's right! Product Designers will now create stories, add them to the drafting board and the roadmap chart ahead of the design sprint kickoff call.

That way, during the sprint kickoff call, we can just pull up the drafting board and go.

cc @marko-lisica @rachaelshaw

Reply:
Also "ESP" => "DRI"